content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -306,6 +306,14 @@ sets this label on that Pod. The value of the label is the name of the Pod being
 See [Pod Name Label](/docs/concepts/workloads/controllers/statefulset/#pod-name-label) in the
 StatefulSet topic for more details.
 
+### scheduler.alpha.kubernetes.io/node-selector {#schedulerkubernetesnode-selector}
+
+Example: `scheduler.alpha.kubernetes.io/node-selector: "name-of-node-selector"`
```

Comment:
My question is shouldn't we graduate this to a stable name like `scheduler.kubernetews.io/node-selector`, then move to the well-known labels, I found a discussion in the past https://github.com/kubernetes/kubernetes/issues/57424, cc @liggitt

Reply:
There's two issues here. We should document the annotation that Kubernetes actually uses, and then later if we fix that annotation to follow project conventions, we should document the new thing. That's for the docs.

We should consider fixing this label to follow project conventions. That's not a docs issue and should be filed against [k/k](https://github.com/kubernetes/kubernetes).

Reply:
Do we have any principle guiding this like feature gates? This has been in alpha for really a long time.

Reply:
If nobody has made the effort to graduate it, it should probably go away.

That said, documenting the current behavior is good.

Reply:
We should deprecate it if we can.

Reply:
by policy, scheduling can simply remove it... it's not on by default and is explicitly alpha

Reply:
Created https://github.com/kubernetes/kubernetes/issues/113474

Reply:
Or we can merge this first until we solve https://github.com/kubernetes/kubernetes/issues/113474